tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6841,12 +6845,45 @@ Status ConvertGraphDefToEngine(
   TF_RETURN_IF_ERROR(statusor.status());
   std::unique_ptr<Converter> converter = std::move(statusor.ValueOrDie());
 
+  GraphDef graph;
+
+  bool apply_layout_optim;
+  Status status =
+      ReadBoolFromEnvVar("TF_TRT_ENABLE_LAYOUT_OPTIMIZER",
+                          /*default_value=*/true, &apply_layout_optim);
+  if (!status.ok()) {
+    LOG(ERROR) << status;
+  }
+  if (cluster != nullptr &&
```

Comment:
If cluster is nullptr, we don't even need to read the env. Can you fix this?
Please also squash your commits into one. I will approve this after the layout PR is merged.

Reply:
Fixed